### PR TITLE
chore(docs): docs with installation instructions/backup instructions

### PR DESCRIPTION
**Pull request by niladrih**
Signed-off-by: Niladri Halder <niladri.halder@mayadata.io>

Add installation instructions for several platforms.
Adds configuration instructions.
Add basic backup/restore instructions.

### CODE REVIEW

**Comment by niladrih** [comment]
This PR does not have any code changes. 

The failing integration tests are due to inadequate timeout duration when checking if finalizers are added to BDCs which don't have finalizers, after openebs-localpv-provisioner pod is restarted. Changes required to fix the failing integration test will be handled in a separate PR.

File: docs/tutorials/hostpath/nodeaffinitylabel.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Use a unique Node-Selector label
+
+Hostpath LocalPV uses the Kubernetes Node label `kubernetes.io/hostname=<node-name>` to uniquely identifly a node.
+In some cases, this label is not persistent across Node reboots and/or Node crashes.
```
Comment (kmova) [inline docs/tutorials/hostpath/nodeaffinitylabel.md]:
```suggestion
In some cases, this label (`hostname`) is not unique across all the nodes in the cluster. This was seen on clusters provisioned with bash across different fault domains.
```

File: docs/quickstart.md [hunk 2]
```diff
@@ -89,27 +75,41 @@ You can provision LocalPV Hostpath volumes dynamically using the default `openeb
...
-      # - name: BasePath     # Use this to set a custom 
-      #   value: /mnt/data   # hostpath directory
-      # - name: NodeAffinityLabel                   # Use this to set a custom 
-      #   value: "openebs.io/custom-node-unique-id" # label for node selection
+          value: "hostpath"
+       #Use this to set a custom
+       # hostpath directory
+       #- name: BasePath
+       #  value: "/mnt/data"
+       #Use this to set a custom
```
Comment (kmova) [inline docs/quickstart.md]: Remove this from the example. Having this confused a lot of users.

File: docs/quickstart.md [hunk 3]
```diff
@@ -89,27 +75,41 @@ You can provision LocalPV Hostpath volumes dynamically using the default `openeb
...
-          value: hostpath
-      # - name: BasePath     # Use this to set a custom 
-      #   value: /mnt/data   # hostpath directory
-      # - name: NodeAffinityLabel                   # Use this to set a custom 
-      #   value: "openebs.io/custom-node-unique-id" # label for node selection
+          value: "hostpath"
+       #Use this to set a custom
+       # hostpath directory
+       #- name: BasePath
+       #  value: "/mnt/data"
```
Comment (kmova) [inline docs/quickstart.md]:
```suggestion
       #  value: "/var/openebs/local"
```

File: docs/tutorials/hostpath/nodeaffinitylabel.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
+# Use a unique Node-Selector label
+
+Hostpath LocalPV uses the Kubernetes Node label `kubernetes.io/hostname=<node-name>` to uniquely identifly a node.
+In some cases, this label (`hostname`) is not unique across all the nodes in the cluster. This was seen on clusters provisioned with bash across different fault domains.
```
Comment (kmova) [inline docs/tutorials/hostpath/nodeaffinitylabel.md]:
```suggestion
In some cases, this label (`hostname`) is not unique across all the nodes in the cluster. This was seen on clusters provisioned with [bosh](https://bosh.io/docs/) across different fault domains.
```

**Review by kmova — APPROVED** [review]

**Pull request merged**